manifests/charts/istio-control/istio-discovery/templates/configmap.yaml

```
@@ -58,12 +58,12 @@
       {{- end }}
       {{- if .Values.global.remotePilotAddress }}
       {{- if .Values.pilot.enabled }}
-      discoveryAddress: {{ printf "istiod-remote.%s.svc" .Release.Namespace }}:15012
+      discoveryAddress: {{ printf "istiod-remote.%s.svc" .Values.global.istioNamespace | default .Release.Namespace }}:15012
```

```suggestion
      discoveryAddress: {{ printf "istiod-remote.%s.svc" (.Values.global.istioNamespace | default .Release.Namespace) }}:15012
```

Not sure that is a style nit or a bug

---

Oh yep, that's a bug; good catch!